apps/dex-explorer/src/pages/tournament/round/index.ts

```diff
@@ -0,0 +1 @@
+export { TournamentRoundPage } from './ui/page';
```

Comment:
suggestion (code structure): move round page to `tournament/ui/round/*`. Don't think we need a separate index file here, exporting page from `tournament/index.ts` would be enough